components/eam/src/physics/cam/gw_common.F90

```diff
@@ -741,5 +745,1387 @@ subroutine gw_drag_prof(ncol, ngwv, src_level, tend_level, do_taper, dt, &
+  real(r8), intent(in) :: dtime
+  real(r8), intent(in) :: nm(state%ncol,pver)   ! midpoint Brunt-Vaisalla frequency
+  !
+  logical , intent(in) :: gwd_ls
+  logical , intent(in) :: gwd_bl
+  logical , intent(in) :: gwd_ss
+  logical , intent(in) :: gwd_fd
+  !tunable parameter from namelist
+  real(r8), intent(in) :: ncleff_ls
+  real(r8), intent(in) :: ncd_bl
+  real(r8), intent(in) :: sncleff_ss
+  !
+  real(r8), intent(out), optional :: utgw(state%ncol,pver)
+  real(r8), intent(out), optional :: vtgw(state%ncol,pver)
+  real(r8), intent(out), optional :: ttgw(state%ncol,pver)
+  !
+  real(r8), intent(out), optional :: dtaux3_ls(pcols,pver)
+  real(r8), intent(out), optional :: dtauy3_ls(pcols,pver)
+  real(r8), intent(out), optional :: dtaux3_bl(pcols,pver)
+  real(r8), intent(out), optional :: dtauy3_bl(pcols,pver)
+  real(r8), intent(out), optional :: dtaux3_ss(pcols,pver)
+  real(r8), intent(out), optional :: dtauy3_ss(pcols,pver)
+  real(r8), intent(out), optional :: dtaux3_fd(pcols,pver)
+  real(r8), intent(out), optional :: dtauy3_fd(pcols,pver)
+  real(r8), intent(out), optional :: dusfc_ls(pcols)
+  real(r8), intent(out), optional :: dvsfc_ls(pcols)
+  real(r8), intent(out), optional :: dusfc_bl(pcols)
+  real(r8), intent(out), optional :: dvsfc_bl(pcols)
+  real(r8), intent(out), optional :: dusfc_ss(pcols)
+  real(r8), intent(out), optional :: dvsfc_ss(pcols)
+  real(r8), intent(out), optional :: dusfc_fd(pcols)
+  real(r8), intent(out), optional :: dvsfc_fd(pcols)
+  !
+  real(r8) :: ztop(pcols,pver)             ! top interface height asl (m)
+  real(r8) :: zbot(pcols,pver)             ! bottom interface height asl (m)
+  real(r8) :: zmid(pcols,pver)             ! middle interface height asl (m)
+  real(r8) :: dz(pcols,pver)               ! model layer height
+  !
+  !real(r8) :: g
+  !pblh input
+  integer  :: pblh_idx     = 0
+  integer  :: kpbl2d_in(pcols)
+  real(r8), pointer :: pblh(:)
+  real(r8) :: dx(pcols),dy(pcols)
+  !needed index
+  integer  :: ncol
+  integer  :: i
+  integer  :: k
+  !local transfer variables
+  real(r8) :: dtaux3_ls_local(pcols,pver)
+  real(r8) :: dtauy3_ls_local(pcols,pver)
+  real(r8) :: dtaux3_bl_local(pcols,pver)
+  real(r8) :: dtauy3_bl_local(pcols,pver)
+  real(r8) :: dtaux3_ss_local(pcols,pver)
+  real(r8) :: dtauy3_ss_local(pcols,pver)
+  real(r8) :: dtaux3_fd_local(pcols,pver)
+  real(r8) :: dtauy3_fd_local(pcols,pver)
+  real(r8) :: dusfc_ls_local(pcols)
+  real(r8) :: dvsfc_ls_local(pcols)
+  real(r8) :: dusfc_bl_local(pcols)
+  real(r8) :: dvsfc_bl_local(pcols)
+  real(r8) :: dusfc_ss_local(pcols)
+  real(r8) :: dvsfc_ss_local(pcols)
+  real(r8) :: dusfc_fd_local(pcols)
+  real(r8) :: dvsfc_fd_local(pcols)
+
+                !
+                ncol=state%ncol
+                !convert heights above surface to heights above sea level
+                !obtain z,dz,dx,dy
+                !ztop and zbot are already reversed, start from bottom to top
+                kpbl2d_in=0_r8
```

Comment:
Can you make the vertical alignment consistent? Breaks in the alignment like this are one of my all-time biggest pet peeves, and have led me personally to incorrectly interpret code, so I can't stand sloppy code like this.